packages/api/app/Models/User.js

```diff
@@ -334,15 +343,23 @@ class User extends Model {
 	 */
 	static scopeWithBookmarksFilters(query, filters) {
 		const technologyId = Number(filters.technologyId);
+		const serviceId = Number(filters.serviceId);
 
 		query.with('bookmarks');
```

Comment:
Acho uma boa, por agora, mudar o nome do relacionamento com as technologies bookmarks para `technologyBookmarks`
```suggestion
		query.with('technologyBookmarks');
```